fabric_plugin/tests/tests.py

```diff
@@ -34,6 +34,7 @@
 from fabric_plugin import tasks
 from cloudify import ctx
 
+from tempfile import mkstemp
```

Comment:
put this import next to the standard library imports in the first part.
if possible, do `import tempfile` and reference `tempfile.mkstemp` later on